.github/workflows/build.yml

```
@@ -397,11 +397,32 @@ jobs:
           name: session-build-${{ matrix.sys.os }}
           path: build/
 
+  scan:
+    name: Scan
+    needs:
+      - os_specific
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download All Build Session Artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: build/
+
+      - name: Scan for CVEs
+        if: runner.os == 'Linux'
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
+          scan-type: rootfs
+          scan-ref: build
+          ignore-unfixed: true
```

```suggestion
          ignore-unfixed: false
```

If I understand this correctly `true` would only report it the first time.

---

No, this will ignore any CVEs that do not have fixes. We currently have an unfixed CVE. Without this option, the check will fail and we will have to force PRs to merge.

---

Ahh I see.. ok yeah that makes sense 👍